src/read.c

```diff
@@ -855,6 +855,7 @@ static avifResult avifDecoderItemMaxExtent(const avifDecoderItem * item, const a
     if (sample->size == 0) {
         return AVIF_RESULT_TRUNCATED_DATA;
     }
+    size_t remainingOffset = sample->offset;
```

Comment:
Declare this as uint64_t, because `sample->offset` is uint64_t:

`    uint64_t remainingOffset = sample->offset;`